### DISCUSSION TITLE
Problem: packages.json refers to a non-existing file

### DISCUSSION THREAD

**Discussion by mkrauser** [renovatebot/renovate#22360] [Request Help]
I have local packages referenced in my package.json like this:

```
"@symfony/ux-autocomplete": "file:vendor/symfony/ux-autocomplete/assets",
```

I run renovate as github-action, but since the composer dependencies of the project are not installed, yarn cannot find the file (or directory) referenced here. That produces the following error:

```
error Package "" refers to a non-existing file '"/tmp/renovate/repos/.../vendor/symfony/ux-autocomplete/assets"'.
```

I've tried to checkout the repo and install composer deps before running renovate, but since the working directory is not mounted in the docker-container this has no effect.

Would it be possible to (optionally) mount the working directory in the docker-container?

That would help a lot.

P.S.: There was also an issue in renovate regarding this problem: https://github.com/renovatebot/config-help/issues/44
I've trie a few things but could not get it to work around this.

**Comment by viceice** [comment]
this is currently not possible
